Kickstarter-iOS/Views/Cells/PledgeContinueCell.swift

```diff
@@ -42,5 +64,11 @@ final class PledgeContinueCell: UITableViewCell, ValueCell {
       |> ksr_constrainViewToMarginsInParent()
 
     self.continueButton.heightAnchor.constraint(greaterThanOrEqualToConstant: Styles.grid(8)).isActive = true
+
+    self.continueButton.addTarget(self, action: #selector(self.continueButtonTapped), for: .touchUpInside)
```

Comment:
I think we're taking the convention of `Type.selector` for these, SwiftFormat does automatically insert `self` but if you replace that with `PledgeContinueCell` it will leave it as that.